rust-toolchain.toml

```diff
@@ -1,4 +1,4 @@
 [toolchain]
-channel = "1.81.0"
+channel = "1.82.0"
```

Comment:
do we need to do this on this PR? I think CI will need to be changed as well... esp. the docker build and publish

Reply:
i will revert this for now